src/Components/Footer/Footer.tsx

```diff
@@ -0,0 +1,86 @@
+import { useContext } from 'react';
+import cn from 'classnames';
+import { deleteTodo } from '../../api/todos';
+import { TodoContext } from '../../Context/TodoContext';
+
+import { Error } from '../../types/Error';
+
+export const Footer: React.FC = () => {
+  const { todos, status, setStatus, deleteTodoLocal, setError, focusInput } =
+    useContext(TodoContext);
+  const activeTodoCount = todos.filter(todo => !todo.completed).length;
+  const completedTodoCount = todos.filter(todo => todo.completed).length;
+
```

Comment:
You can make two values( active, completed) with one array iteration using the reduce method. ( not critical)